packages/vendor-core/scss/vendor-core.scss

```diff
@@ -10,6 +10,7 @@
 @import '~select2/select2.css';
 @import "~dsmorse-gridster/dist/jquery.gridster";
 @import "~datatables.net-bs/css/dataTables.bootstrap.css";
+@import '~patternfly-react/dist/sass/_inline-edit.scss';
```

Comment:
Are you sure we need it?
Looks like we are already pulling this file next line when importing `@import '~patternfly-react/dist/sass/_patternfly-react.scss';`

Look like this file contains the `inline-edit` partial:
https://github.com/patternfly/patternfly-react/blob/master/packages/patternfly-3/patternfly-react/sass/patternfly-react/_patternfly-react.scss

Reply:
you are right... thanks for your help!